components/button/index.css

```diff
@@ -229,18 +249,28 @@ a.spectrum-Button {
   }
 
   &:disabled,
-  &.is-disabled {
+  &.is-disabled,
+  &[pending],
+  &.is-pending {
     background-color: var(--highcontrast-button-background-color-disabled, var(--mod-button-background-color-disabled, var(--spectrum-button-background-color-disabled)));
     border-color: var(--highcontrast-button-border-color-disabled, var(--mod-button-border-color-disabled, var(--spectrum-button-border-color-disabled)));
     color: var(--highcontrast-button-content-color-disabled, var(--mod-button-content-color-disabled, var(--spectrum-button-content-color-disabled)));
   }
+
+  &[pending],
+  &.is-pending {
+    cursor: default;
+  }
 }
 
 /* windows high contrast mode over-writes for accent variant */
 @media (forced-colors: active) {
   .spectrum-Button {
     --highcontrast-button-content-color-disabled: GrayText;
     --highcontrast-button-border-color-disabled: GrayText;
+    --mod-progress-circle-track-border-color: CanvasText;
```

Comment:
Should this double border around the progress circle be set to `ButtonText` to ensure contrast when on top of `ButtonFace`?